NREL/ThermalNetwork
# Additional output file for GMT to build multi-ghe Modelica models

#### Any background context you want to provide?
Multi-ghe districts have additional information that is needed to build a Modelica model appropriately
#### What does this PR accomplish?
Generates additional json file with loop order information for the GMT & REopt to consume
#### How should this be manually tested?
- Run [the 13-building test](https://github.com/NREL/ThermalNetwork/blob/multi-ghe/thermalnetwork/tests/test_network.py#L82)
- Confirm `_loop_order.json` file is created/updated next to the sys-param file
    - Since this is generated by the code every time the test is run, should it be git-ignored?
- Confirm the test passes
- Confirm the contents of those files hold the appropriate information
#### What are the relevant tickets?

#### Screenshots (if appropriate)
